_includes/v23.1/known-limitations/cdc-queries.md

```diff
@@ -0,0 +1,9 @@
+- You can only apply CDC queries on a single table in each statement.
```

Comment:
This is not a limitation per say, but I think we need to mention that quite many functions depend on session data -- session that can specify and/or change default timezone location, how strings are collated, etc.  The session data is saved at the time of the changefeed creation -- and that session data is used when evaluating those functions.

Reply:
Added a bullet on this under the function support section,